src/javascript/app/pages/trade/purchase.js

```diff
@@ -105,6 +105,11 @@ const Purchase = (() => {
                     BinarySocket.wait('get_account_status').then(response => {
```

Comment:
why do we need to get the `get_account_status` response to show the message?

Reply:
If i put it outside, the message doesnt get displayed.

Reply:
you can add this in line 104
```suggestion
else if (/InsufficientBalance/.test(error.code)) {
                    confirmation_error.setVisibility(1);
                    const redirect = '<a href="https://www.binary.com/en/terms-and-conditions.html?anchor=trading-limits#legal-binary" target="_blank" rel="noopener noreferrer">';
                    const redirect_close = '</a>';
                    let message = localize('No further trading is allowed on this contract type for the current trading session. For more info, refer to our [_1]terms and conditions[_2].',[redirect, redirect_close]);
                    CommonFunctions.elementInnerHtml(confirmation_error, message);
                }```

Reply:
```suggestion
                   else if (/CompanyWideLimitExceeded/.test(error.code)) {
                    confirmation_error.setVisibility(1);
                    const redirect = '<a href="https://www.binary.com/en/terms-and-conditions.html?anchor=trading-limits#legal-binary" target="_blank" rel="noopener noreferrer">';
                    const redirect_close = '</a>';
                    let message = localize('No further trading is allowed on this contract type for the current trading session. For more info, refer to our [_1]terms and conditions[_2].',[redirect, redirect_close]);
                    CommonFunctions.elementInnerHtml(confirmation_error, message);
```